src/openrct2/world/Climate.h

```diff
@@ -21,20 +22,26 @@ enum class ClimateType : uint8_t
     Count
 };
 
-enum WEATHER
+enum class WeatherType : uint8_t
 {
-    WEATHER_SUNNY,
-    WEATHER_PARTIALLY_CLOUDY,
-    WEATHER_CLOUDY,
-    WEATHER_RAIN,
-    WEATHER_HEAVY_RAIN,
-    WEATHER_THUNDER,
-    WEATHER_SNOW,
-    WEATHER_HEAVY_SNOW,
-    WEATHER_BLIZZARD,
-    WEATHER_COUNT,
+    Sunny,
+    PartiallyCloudy,
+    Cloudy,
+    Rain,
+    HeavyRain,
+    Thunder,
+    Snow,
+    HeavySnow,
+    Blizzard,
+    Count
 };
 
+constexpr WeatherType weatherTypes[EnumValue(WeatherType::Count)] = { WeatherType::Sunny,     WeatherType::PartiallyCloudy,
```

Comment:
I don't see the reason to have this and runs in contrast to the rest of the codebase. Just use EnumValue where required.

Reply:
It was there because I didn't know you could do this: `WeatherType{ <uint8_t argument> }`. It is fixed now